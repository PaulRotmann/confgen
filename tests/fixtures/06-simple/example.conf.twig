# just a sample
timeout = {{ data.timeout }};

[all]
{% for value in data.values %}
add {{ value }};
{% endfor %}
