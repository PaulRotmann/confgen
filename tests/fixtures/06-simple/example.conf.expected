# just a sample
timeout = 20;

[all]
add 10;
add 12;
